util/src/main/java/io/kubernetes/client/informer/cache/ProcessorListener.java

```diff
@@ -90,9 +87,7 @@ public void add(Notification<ApiType> obj) {
     if (obj == null) {
       return;
     }
-    if (!this.queue.offer(obj)) {
-      log.warn("notification queue full!");
-    }
+    this.queue.add(obj);
```

Comment:
minimun change. it doesnt really make a difference tho.

Reply:
Yeah, but the code is less...the log is redundant as the capacity is unlimited.